src/components/use-collapsible/use-collapsible.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
```

Comment:
Why do we put this into components? Would a hooks folder be more apt?
We could also call it useToggle I guess.

Reply:
Yeah, I agree with Tobi

Reply:
ya good call 🎈